x-pack/plugins/cases/public/components/all_cases/use_all_cases_state.test.tsx

```diff
@@ -74,225 +48,142 @@ describe('useAllCasesQueryParams', () => {
       wrapper: ({ children }) => <TestProviders>{children}</TestProviders>,
     });
 
-    expect(result.current.queryParams).toStrictEqual(DEFAULT_QUERY_PARAMS);
-    expect(result.current.filterOptions).toStrictEqual(DEFAULT_FILTER_OPTIONS);
-  });
-
-  it('updates localstorage with default values on first run', () => {
```

Comment:
local storage is not updated on the first run. it is updated when the user interacts with the filters and the URL changes.

Reply:
This changed in the latest commit right? Is the test still relevant?

Reply:
Good question. We do not update the local storage with default values. We only update the local storage if the URL is not empty. The test that tests this behavior is called `updates the local storage when navigating to a URL and the query params are not empty`.